xbmc/pvr/PVRDatabase.cpp

```diff
@@ -45,7 +45,7 @@ void CPVRDatabase::CreateTables()
 {
   CLog::Log(LOGINFO, "PVR - %s - creating tables", __FUNCTION__);
 
-  CLog::Log(LOGDEBUG, "PVR - %s - creating table 'channels'", __FUNCTION__);
+  CLog::Log(LOGDEBUG, "CPVRDatabase - %s - creating table 'channels'", __FUNCTION__);
```

Comment:
dvdplayer (I think that's the one you meant) is just 1 class. most of the rest of the code just uses the name of the component, if it uses any prefix at all.

logging pointless things (imo) just slows things down, especially on low powered devices.